torch_xla/core/xla_model.py

```diff
@@ -140,6 +140,7 @@ def get_ordinal(defval=0):
   if runtime.using_pjrt():
     return runtime.global_ordinal()
 
+  print("Please set PJRT_DEVICE")
```

Comment:
I think you can just error out here. @will-cromar to confirm

Reply:
thanks for pointing, test in https://github.com/pytorch/xla/pull/5228

Reply:
Thanks, updated